### DISCUSSION TITLE
Suggestion: When typing anything in the branch name, there should be a button for it.

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#5624] [Codespaces]
In VSCode desktop:
![image](https://user-images.githubusercontent.com/50248166/133136221-b91f5f0c-5a47-446c-94c7-6d5eb5683998.png)
In Codespaces:
![image](https://user-images.githubusercontent.com/50248166/133136244-92c9405b-d8a8-479c-a687-6f9fc7a7602d.png)

I am suggesting that in Codespaces the behaviour is like it is in vscode desktop.

**Comment by bamurtaugh** [comment]
Thanks for opening @Milo123459. I just tested this behavior in Codespaces in the web, and I am shown the options to create a new branch:

![image](https://user-images.githubusercontent.com/25310137/136620280-09b0b090-a4cb-41fd-b277-8016f50cdb4a.png)

Are you still able to repro the original issue?

**Reply by Milo123459** [reply]
This isn't proper codespaces, it's the "pressing . to open" codespaces. But still, this bug happens.

**Reply by bamurtaugh** [reply]
Thanks for clarifying. I think that'd stem from the GitHub Repositories extension, and it'd make sense to file an issue there. Thank you! https://github.com/microsoft/vscode-remote-repositories-github/issues
